src/api/index.yaml

```diff
@@ -61,6 +61,7 @@ definitions:
       - supportedEventTypes
       - startDate
       - endDate
+      - portsDataset
```

Comment:
I would call this `portsGCSPath` or something more explicit.

Reply:
Good catch! thanks and it makes me think one question:
is this needed https://github.com/GlobalFishingWatch/api-settings/pull/8/files#diff-d5fb9b15dd424a639a4d4c7e12e6141eR2 anymore ? or would you go for a simple request for the public url ?